vnext/ReactUWP/Views/CheckboxViewManager.cpp

```diff
@@ -32,23 +30,28 @@ class CheckBoxShadowNode : public ShadowNodeBase {
  private:
   static void
   OnCheckedChanged(IReactInstance &instance, int64_t tag, bool newValue);
+
+  winrt::CheckBox::Checked_revoker m_checkBoxCheckedRevoker{};
+  winrt::CheckBox::Unchecked_revoker m_checkBoxUncheckedRevoker{};
 };
 
 void CheckBoxShadowNode::createView() {
   Super::createView();
 
   auto checkbox = GetView().as<winrt::CheckBox>();
   auto wkinstance = GetViewManager()->GetReactInstance();
-  checkbox.Checked([=](auto &&, auto &&) {
-    auto instance = wkinstance.lock();
-    if (!m_updating && instance != nullptr)
-      OnCheckedChanged(*instance, m_tag, true);
-  });
-  checkbox.Unchecked([=](auto &&, auto &&) {
-    auto instance = wkinstance.lock();
-    if (!m_updating && instance != nullptr)
-      OnCheckedChanged(*instance, m_tag, false);
-  });
+  m_checkBoxCheckedRevoker =
+      checkbox.Checked(winrt::auto_revoke, [=](auto &&, auto &&) {
```

Comment:
Use explicit captures rather than [=]

Reply:
this is not the only one, I create bug #2748 to tack it

---
In reply to: [301721470](https://github.com/microsoft/react-native-windows/pull/2746#discussion_r301721470) [](ancestors = 301721470)

Reply:
It's good to have a bug to fix all the other [=] already in the code base, but you should fix this new one before checking this in.

Reply:
Nevermind, it's not new :)